rfc/text/advanced/ap_50_payload_passthru_mode.md

```diff
@@ -0,0 +1,422 @@
+        INVOCATION.Details.ppt_serializer|string
+        INVOCATION.Details.ppt_cipher|string
+        INVOCATION.Details.ppt_keyid|string
+        ---
+        YIELD.Options.ppt_scheme|string
+        YIELD.Options.ppt_serializer|string
+        YIELD.Options.ppt_cipher|string
+        YIELD.Options.ppt_keyid|string
+        ---
+        RESULT.Details.ppt_scheme|string
+        RESULT.Details.ppt_serializer|string
+        RESULT.Details.ppt_cipher|string
+        RESULT.Details.ppt_keyid|string
+        ---
+        ERROR.Details.ppt_scheme|string
+        ERROR.Details.ppt_serializer|string
+        ERROR.Details.ppt_cipher|string
+        ERROR.Details.ppt_keyid|string
+
+{align="left"}
+        PUBLISH.Options.ppt_scheme|string
+        PUBLISH.Options.ppt_serializer|string
+        PUBLISH.Options.ppt_cipher|string
+        PUBLISH.Options.ppt_keyid|string
+        ---
+        EVENT.Details.ppt_scheme|string
+        EVENT.Details.ppt_serializer|string
+        EVENT.Details.ppt_cipher|string
+        EVENT.Details.ppt_keyid|string
+        ---
+        ERROR.Options.ppt_scheme|string
+        ERROR.Options.ppt_serializer|string
+        ERROR.Options.ppt_cipher|string
+        ERROR.Options.ppt_keyid|string
+
+
+**ppt_scheme Attribute**
+
+The `ppt_scheme` identifies the Key Management Schema. It is a required string attribute. This attribute can contain 
+the name or identifier of a key management provider that is known to the target peer, so it can be used to obtain 
+information about encryption keys. A *Router* can recognize that `Payload Passthru Mode` is in use
+by checking the existence and non-empty value of this attribute within the options of `CALL`, `PUBLISH` and `YIELD` 
+messages.
+
+**ppt_serializer Attribute**
+
+The `ppt_serializer` attribute is optional. It specifies what serializer was used to encode the payload.
+It can be a value a such as `mqtt`, `amqp`, `stomp` to indicate that the incoming data is tunneling through such 
+technologies, or it can be ordinary `json`, `msgpack`, `cbor`, `flatbuffers` data serializers.
+
+**ppt_cipher Attribute**
+
+The `ppt_cipher` attribute is optional. It is required if the payload is encrypted. This attribute specifies the 
+cryptographic algorithm that was used to encrypt the payload. It can be `xsalsa20poly1305`, `aes256gcm` for now.
+
+**ppt_keyid Attribute**
+
+The `ppt_keyid` attribute is optional. This attribute can contain the encryption key id that was used to encrypt 
+the payload. The `ppt_keyid` attribute is a string type. The value can be a hex-encoded string, URI, DNS name, 
+Ethereum address, UUID identifier - any meaningful value which allows the target peer to choose a private key 
+without guessing. The format of the value may depend on the `ppt_scheme` attribute.
+
+**ppt_ Predefined Schemes**
+
+| Attribute      | Required? | Predefined Scheme (MQTT)    | Predefined Scheme (End-to-End Encryption) | Custom Scheme Example |
```

Comment:
It's not clear if MQTT and EtoE are just examples or are mandated by the WAMP spec.

Reply:
This would be best split into separate tables to fit the page width in RFC text format.